app/views/static/home.erb

```diff
@@ -27,19 +27,15 @@
                     <input class="btn btn-outline-danger" type="submit" value="Answer">
                 </form>
 
-                <form action="/upvotes/<%= q.id %>" method="post">
-                    <button class="btn btn-secondary like" type="submit" name="up_vote" style="background-color:#1B6AA5; width: 70px">
+                    <a href="/upvotes/<%= q.id %>" class="btn btn-secondary like" type="submit" name="up_vote" style="background-color:#1B6AA5; width: 70px">
                         <i class="fa fa-thumbs-up" aria-hidden="true" style="margin-right:10px"></i>
-                       <%= q.upvotes(q.id) %>
```

Comment:
Removing these params wasn't to fix voting, it was just to fix something that was a little messy.